README.md

```diff
@@ -125,48 +124,51 @@ $givenNames         = $profile->getGivenNames()->getValue();
 
 $fullName           = $profile->getFullName()->getValue();
 
-$dateOfBirth        = $profile->getDateOfBirth()->getValue();
+$dateOfBirth        = $profile->getDateOfBirth()->getValue(); // DateTime Object
 
 $gender             = $profile->getGender()->getValue();
 
 $nationality        = $profile->getNationality()->getValue();
 
 $phoneNumber        = $profile->getPhoneNumber()->getValue();
 
-$selfie             = $profile->getSelfie()->getValue();
+$selfie             = $profile->getSelfie()->getValue(); // Yoti\Entity\Image Object
 
 $emailAddress       = $profile->getEmailAddress()->getValue();
 
 $postalAddress      = $profile->getPostalAddress()->getValue();
 
-$ageVerified        = $profile->getAgeCondition()->getValue();
+$ageVerifications   = $profile->getAgeVerifications(); // array of AgeVerification object, e.g ['age_over:18' => new AgeVerification(),]
+
+$ageOverXX          = $profile->findAgeOverVerification($xx); // AgeVerification Object (please see API Coverage for details), Where $xx is the age value, e.g 50
+
+$ageUnderXX         = $profile->findAgeUnderVerification($xx); // AgeVerification Object (please see API Coverage for details), Where $xx is the age value, e.g 18
 
-$verifiedAge        = $profile->getVerifiedAge()->getValue();
 ```
 
 ## Handling Users
 
-When you retrieve the user profile, you receive a user ID generated by Yoti exclusively for your application.
+When you retrieve the user profile, you receive a rememberMe ID generated by Yoti exclusively for your application.
```

Comment:
I think this should either be `Remember Me Id` or `RememberMeId` (2 spaces or no spaces). It looks like it's called `rememberMeId` in the dev docs, so I would lean towards that.

Reply:
👍